static/js/src/advantage/contracts-api.js

```diff
@@ -37,17 +46,24 @@ export async function getRenewal(renewalID) {
 }
 
 export async function getCustomerInfo(accountId) {
-  let response = await fetch(`/advantage/customer-info/${accountId}`, {
-    cache: "no-store",
-  });
+  const queryString = window && window.location ? window.location.search : "";
+
+  let response = await fetch(
+    `/advantage/customer-info/${accountId}${queryString}`,
+    {
+      cache: "no-store",
+    }
+  );
 
   let data = await response.json();
   return data;
 }
 
 export async function postInvoiceID(transactionType, transactionID, invoiceID) {
+  const queryString = window && window.location ? window.location.search : "";
```

Comment:
Is it really feasible that `window` or `window.location` are not available?

Reply:
Yeah I don't think so now that I think on it. Just a habit from my Node days